stubs/core.stub

```diff
@@ -1,5 +1,13 @@
 <?php
 
+/**
+ * @param mixed $object_or_class
```

Comment:
couldn't we even use a conditional parameter type here?

```suggestion
 * @param ($allow_string is false ? object : object|class-string) $object_or_class
```

Reply:
oh my.. now I get what you was telling me in https://github.com/phpstan/phpstan-src/pull/1306#issuecomment-1125009375 .. will send a 2nd PR ..